src/vivarium_gates_child_iv_iron/data/loader.py

```diff
@@ -335,3 +342,121 @@ def load_pem_csmr(key: str, location: str) -> pd.DataFrame:
 def load_pem_restrictions(key: str, location: str) -> pd.DataFrame:
     metadata = load_metadata(data_keys.PEM.RESTRICTIONS, location)
     return metadata
+
+
+def load_lbwsg_exposure(key: str, location: str) -> pd.DataFrame:
+    if key != data_keys.LBWSG.EXPOSURE:
+        raise ValueError(f'Unrecognized key {key}')
+
+    key = EntityKey(key)
+    entity = utilities.get_entity(key)
+    data = utilities.get_data(key, entity, location, gbd_constants.SOURCES.EXPOSURE, 'rei_id',
+                              metadata.AGE_GROUP.GBD_2019_LBWSG_EXPOSURE, metadata.GBD_2019_ROUND_ID, 'step4')
+    data = data[data['year_id'] == 2019].drop(columns='year_id')
+    data = utilities.process_exposure(data, key, entity, location, metadata.GBD_2019_ROUND_ID,
+                                      metadata.AGE_GROUP.GBD_2019_LBWSG_EXPOSURE | metadata.AGE_GROUP.GBD_2019)
+    data = data[data.index.get_level_values('year_start') == 2019]
+    return data
+
+
+def load_lbwsg_rr(key: str, location: str) -> pd.DataFrame:
+    if key != data_keys.LBWSG.RELATIVE_RISK:
+        raise ValueError(f'Unrecognized key {key}')
+
+    key = EntityKey(key)
+    entity = utilities.get_entity(key)
+    data = utilities.get_data(key, entity, location, gbd_constants.SOURCES.RR, 'rei_id',
+                              metadata.AGE_GROUP.GBD_2019_LBWSG_RELATIVE_RISK, metadata.GBD_2019_ROUND_ID, 'step4')
+    data = data[data['year_id'] == 2019].drop(columns='year_id')
+    data = utilities.process_relative_risk(data, key, entity, location, metadata.GBD_2019_ROUND_ID,
+                                           metadata.AGE_GROUP.GBD_2019, whitelist_sids=True)
+    data = (
+        data.query('year_start == 2019')
+        .droplevel(['affected_entity', 'affected_measure'])
+    )
+    data = data[~data.index.duplicated()]
+    return data
+
+
+def load_lbwsg_interpolated_rr(key: str, location: str) -> pd.DataFrame:
+    if key != data_keys.LBWSG.RELATIVE_RISK_INTERPOLATOR:
+        raise ValueError(f'Unrecognized key {key}')
+
+    rr = get_data(data_keys.LBWSG.RELATIVE_RISK, location).reset_index()
+    rr['parameter'] = pd.Categorical(rr['parameter'], [f'cat{i}' for i in range(1000)])
+    rr = (
+        rr.sort_values('parameter')
+        .set_index(metadata.ARTIFACT_INDEX_COLUMNS + ['parameter'])
+        .stack()
+        .unstack('parameter')
+        .apply(np.log)
+    )
+
+    # get category midpoints
+    def get_category_midpoints(lbwsg_type: str) -> pd.Series:
```

Comment:
Changed function to take string argument for LBWSG subrisk type instead of taking class type.